Signal/src/ViewControllers/ThreadSettings/OWSConversationSettingsViewController.m

```diff
@@ -363,62 +363,63 @@ - (void)updateTableContents
                                  }]];
     }
 
```

Comment:
I noticed that layout was broken in conversation settings view.  Mostly these are whitespace changes courtesy of clang-format.